runtests.sh

```diff
@@ -13,8 +13,22 @@ echo "Running flake8"
 flake8 --show-source || { echo "flake8 errors found!"; exit 1; }
 
 echo "Running isort"
-isort --check-only --diff --quiet \
+isort \
```

Comment:
Would you be able to move the `isort` related changes into their own PR? (If not too much trouble).